CHANGELOG.md

```diff
@@ -1,9 +1,35 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+This project adheres to [Semantic Versioning](http://semver.org/)
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## 1.2.0 2017-05-13
+## [Unreleased]
+
+## [1.3.0] - 2018-02-04
+### Added
+- Initial support for OpenBSD! (@smbambling)
+- Ubuntu now get's `apt-transport-https` installed to support HTTPS repos. (@kevit)
+- Default to HTTPS APT repo's.  @jaredledvina
+- Ubuntu - Redis now always binds to `0.0.0.0` to ensure it's accessible. (@tculp)
+- Allow for configuring when a node gets the `sensu-client` config file. (@tculp)
+- Split up the variables used to determine if a host gets rabbitmq/redis for more flexibility in deployments. (@tculp)
```

Comment:
Are all the old options still functional? If not this is also a breaking change.

Reply:
Related change: https://github.com/sensu/sensu-ansible/commit/347afa18fd3a25343e6d2a17d9c5e525a811e92c

Crap, yep, version 2.0.0 here we come.

Reply:
Yup, it's hard to version software...